src/runtime-rs/crates/resource/src/manager.rs

```diff
@@ -50,9 +50,12 @@ impl ResourceManager {
         inner.config()
     }
 
-    pub async fn prepare_before_start_vm(&self, device_configs: Vec<ResourceConfig>) -> Result<()> {
+    pub async fn prepare_before_or_after_start_vm(
+        &self,
+        device_configs: Vec<ResourceConfig>,
+    ) -> Result<()> {
         let mut inner = self.inner.write().await;
-        inner.prepare_before_start_vm(device_configs).await
+        inner.prepare_before_or_after_start_vm(device_configs).await
```

Comment:
I think there is no need to change the name.

Reply:
After the VM starts, I think it's inappropriate to call it `prepare_before_start_vm`

Reply:
fixed

Reply:
The function name `prepare_before_or_after_start_vm` is not updated.

Reply:
fixed